CrudMicroservice.Domain/ApiModel/ResponseApiModels/EuroProtocolFullResponseModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CrudMicroservice.Data.Entities;
+using System.Collections.Generic;
+
+namespace CrudMicroservice.Domain.ApiModel.ResponseApiModels
+{
+    public class EuroProtocolFullResponseModel
+    {
+        public EuroProtocolSimpleResponseModel EuroProtocolSimple { get; set; }
```

Comment:
it should contain 'EuroProtocolResponseModel', not 'EuroProtocolSimpleResponseModel', and property should be named just as 'EuroProtocol'